frontend/src/container/MetricsTable/Metrics.test.tsx

```diff
@@ -1,34 +1,47 @@
-import { render, RenderResult, screen, waitFor } from '@testing-library/react';
-import { ReactElement } from 'react';
-import { Provider } from 'react-redux';
+import { render, screen, waitFor } from '@testing-library/react';
 import { BrowserRouter } from 'react-router-dom';
-import {
-	combineReducers,
-	legacy_createStore as createStore,
-	Store,
-} from 'redux';
 
-import { InitialValue } from '../../store/reducers/metric';
 import Metrics from './index';
 
-const rootReducer = combineReducers({
-	metrics: (state = InitialValue) => state,
-});
-
-const mockStore = createStore(rootReducer);
+jest.mock('react-router-dom', () => ({
+	...jest.requireActual('react-router-dom'),
+	useLocation: (): { pathname: string } => ({
+		pathname: 'localhost:3001/services/',
+	}),
+}));
 
-const renderWithReduxAndRouter = (mockStore: Store) => (
-	component: ReactElement,
-): RenderResult =>
-	render(
-		<BrowserRouter>
-			<Provider store={mockStore}>{component}</Provider>
-		</BrowserRouter>,
-	);
+const services = [
```

Comment:
put this one in mocks

Reply:
This data is directly send to the component, can't be mock